pkg/sql/logictest/testdata/logic_test/cluster_locks

```diff
@@ -145,6 +145,27 @@ test          public      t           ·                   Exclusive       Repli
 
 user root
 
+query TTTTTTBB colnames,retry
+SELECT database_name, schema_name, table_name, lock_key_pretty, lock_strength, durability, granted, contended FROM crdb_internal.cluster_locks WHERE database_name='test'
```

Comment:
can you add a test for `table_id` and `table_name`?

to get the tableid, you can probably do `WHERE table_id = 'table_name'::regclass::oid::int`.